README.md

```diff
@@ -66,6 +68,31 @@ Once a given scenario configured, it seemed useful to choose how the multi-partn
 
 Finally, with given scenarios and multi-partner learning approaches, we can address contributivity measurement approaches. See below sections [Configuration of contributivity measurement methods to be tested](./mplc/docs/documentation.md#scenario-parameters) and [Contributivity measurement approaches studied and implemented](./mplc/docs/documentation.md#configuration-of-contributivity-measurement-methods-to-be-tested).
 
+### Installation 
+
+##### Using pip
+
+```bash
+
+$ pip install mplc
+
+```
+
+This installs the last packaged version on pypi. If your platform doesn't have a ready package, please open an issue to let us know.
```

Comment:
I don't understand what "If your platform doesn't have a ready package" means?

Reply:
I meant if you got trouble installing the library 
For instance, with tensorflow... But as nobody, whether they tried on a linux, mac or windose had issues, it may be not relevant